### ISSUE TITLE
Jquery problems

### ISSUE THREAD

**Issue by oranze** [muicss/loadjs#80]
When loading Jquery and like example Jquery Cookie plugin via loader scripts not working. $ is undefined ... I think something with name spaces

**Comment by amorey** [comment]
By default LoadJS loads files with `async=true` which will cause problems for the plugin if it loads before jquery. To avoid this you can use `async=false`.

**Comment by oranze** [comment]
I'm already solv this problem, but $.cookie just wont work... interesting what with delay 100ms it starting...

**Comment by amorey** [comment]
Can you share some example code? It doesn't sound like the problem is with LoadJS but if it is I can help you debug it.

Have you tried wrapping your jQuery code in `$.ready()`?

**Comment by oranze** [comment]
Yes... like I sad help delay timeout between run $.cookie in template. may be this is $.cookie plugin bug, but may be this is Gecko engine bug... I get same bug in any Gecko browser. IE not tested yet

**Comment by amorey** [comment]
Here's a fiddle that loads jQuery and jQuery-cookie using LoadJS without a delay:
https://jsfiddle.net/muicss/bd4L9omr/

Let me know if you're able to figure out the problem.
